main.py

```diff
@@ -13,9 +15,45 @@
                                              initial_distribution=v,
                                              num_stages=N, stopping_time=tau).create()
 
-tree.bulls_eye_plot(dot_size=5, radius=300)
+# tree.bulls_eye_plot(dot_size=5, radius=300)
+#
+# tree.set_data_at_node(4, {"a": 1})
+# print(sum(tree.probability_of_node(tree.nodes_at_stage(8))))
+#
+# print(tree)
+
+# RAOCP generation -----------------------------------------------------------------------------------------------------
+
+root_state = np.array([[1],
```

Comment:
Isn't the `root_state` a parameter of the optimisation problem? Why should it be fixed?